lib/model/Command.js

```diff
@@ -31,6 +31,15 @@ const Command = new Schema({
     value: Object,
     service: { type: String, lowercase: true },
     subservice: String,
+    execTs: { type: Date },
+    status: Object,
+    info: Object,
+    onDelivered: Object,
+    onOk: Object,
+    onError: Object,
+    onInfo: Object,
+    cmdExecution: Object,
+    dateExpiration: { type: Date },
```

Comment:
Now that PR https://github.com/telefonicaid/iotagent-node-lib/pull/1717 has been merged... should these fields also added to database model doc?